test/lib.sh

```diff
@@ -183,16 +183,28 @@ BUSER=clicon
 
 : ${clixon_cli:=clixon_cli}
 
-: ${clixon_netconf:=$(which clixon_netconf)}
+: ${clixon_netconf:=clixon_netconf}
 
 : ${clixon_restconf:=clixon_restconf}
 
 : ${clixon_backend:=clixon_backend}
 
+: ${clixon_util_socket:=clixon_util_socket}
+
 : ${clixon_snmp:=$(type -p clixon_snmp)}
 
 : ${clixon_snmp_pidfile:="/var/tmp/clixon_snmp.pid"}
 
+: ${_ALREADY_HERE:=0}
+
+if [ -n "$CLICON_GROUP" ] && [ $_ALREADY_HERE -eq 0 ]; then
```

Comment:
This has the effect of introducing double sudo:s, see last line: sudo -u root sudo clixon_restconf
```
olof@alarik> ./test_restconf_callhome.sh 
Running test_restconf_callhome.sh
Generating a RSA private key
...................+++++
.......+++++
writing new private key to '/var/tmp/./test_restconf_callhome.sh/certs/ca_key.pem'
-----
.......................................................+++++
................+++++
Signature ok
subject=CN = localhost, emailAddress = olof@hagsand.se, O = Clixon, L = Stockholm, C = SE
Getting CA Private Key
..........+++++
............................................+++++
Signature ok
subject=CN = andy, emailAddress = andy@foo.bar, O = Clixon, L = Stockholm, C = SE
Getting CA Private Key
Test 1(1) [test params: -f /var/tmp/./test_restconf_callhome.sh/conf_yang.xml]
Test 2(2) [kill old backend]
Sep  5 11:07:03: Killing old daemon with pid: 2199164
Test 3(3) [start backend  -s startup -f /var/tmp/./test_restconf_callhome.sh/conf_yang.xml]
Test 4(4) [wait backend]
Test 5(5) [kill old restconf daemon]
Test 6(6) [start restconf daemon]
sudo -u root -s sudo -g clicon clixon_restconf  -D 0 -f /var/tmp/./test_restconf_callhome.sh/conf_yang.xml -D 1 -l s
```

Reply:
Fixed.